app/controllers/FileValidationController.scala

```diff
@@ -86,9 +86,9 @@ class FileValidationController @Inject() (
                           for {
                             updatedAnswers <- Future.fromTry(UserAnswers(request.userId).set(InvalidXMLPage, fileName))
                             _              <- sessionRepository.set(updatedAnswers)
-                          } yield Redirect(navigator.nextPage(InvalidXMLPage, NormalMode, updatedAnswers))
+                          } yield Redirect(routes.FileErrorController.onPageLoad())
 
-                        case _ =>
+                        case e =>
```

Comment:
willing changing this throw up a warn?

Reply:
it wouldn't because it's still a catch all, but it was just left over from when we were printing out the error to trace an issue and isn't needed now so I've removed it.